packages/sublime_merge.rb

```diff
@@ -0,0 +1,41 @@
+require 'package'
+
+class Sublime_merge < Package
+  description 'A snappy UI, three-way merge tool,side-by-side diffs, syntax highlighting, and more.'
+  homepage 'https://www.sublimemerge.com/'
+  version 'Build 1065'
+  case ARCH
+  when 'x86_64'		  
+	source_url 'https://download.sublimetext.com/sublime_merge_build_1065_x64.tar.xz'
+	source_sha256 'ebe9029ea71543a101dcfd09c5540759759c1e33fea31d4b98256803b7ceaf97'
+  else
+    puts 'Merge is not supported on your system yet.'
+  end
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'xdg_base'
```

Comment:
Why do we need xdg_base?

Reply:
Just a leftover from Code that I was debating whether it would be useful or not. I will remove it and re-add it at a later time if an issue arises.